diff -up WRF-WPS-3.6.1/WRFV3/share/mediation_feedback_domain.F.serial WRF-WPS-3.6.1/WRFV3/share/mediation_feedback_domain.F
--- WRF-WPS-3.6.1/WRFV3/share/mediation_feedback_domain.F.serial	2014-07-18 10:50:09.000000000 -0600
+++ WRF-WPS-3.6.1/WRFV3/share/mediation_feedback_domain.F	2014-10-08 12:28:36.000000000 -0600
@@ -6,7 +6,9 @@ SUBROUTINE med_feedback_domain ( parent_
    USE module_domain
    USE module_configure
    USE module_intermediate_nmm
+#ifdef NMM_FIND_LOAD_IMBALANCE
    USE module_dm, only: local_communicator
+#endif
    IMPLICIT NONE
    TYPE(domain), POINTER :: parent_grid , nested_grid
    TYPE(domain), POINTER :: grid
