diff -up WRF-WPS-3.8.1-intel-2016-openmpi-1.10-3.8.1/WRFV3/external/io_netcdf/makefile.netcdf WRF-WPS-3.8.1-intel-2016-openmpi-1.10-3.8.1/WRFV3/external/io_netcdf/makefile
--- WRF-WPS-3.8.1-intel-2016-openmpi-1.10-3.8.1/WRFV3/external/io_netcdf/makefile.netcdf	2014-07-12 17:39:10.000000000 -0600
+++ WRF-WPS-3.8.1-intel-2016-openmpi-1.10-3.8.1/WRFV3/external/io_netcdf/makefile	2016-09-27 08:48:34.677166389 -0600
@@ -4,8 +4,8 @@ OBJSL   = wrf_io.o field_routines.o modu
 OBJS    = $(OBJSL)
 CODE    = ext_ncd_get_dom_ti.code ext_ncd_get_var_td.code ext_ncd_get_var_ti.code ext_ncd_put_dom_ti.code ext_ncd_put_var_td.code ext_ncd_put_var_ti.code transpose.code 
 FFLAGS  =  $(FCFLAGS) -I$(NETCDFPATH)/include -I../ioapi_share
-LIBS    = $(LIB_LOCAL) -L$(NETCDFPATH)/lib -lnetcdf
-LIBFFS  = $(LIB_LOCAL) -L$(NETCDFPATH)/lib -lnetcdff -lnetcdf $(NETCDF4_DEP_LIB)
+LIBS    = $(LIB_LOCAL) -L$(NETCDF_LIB) -lnetcdf
+LIBFFS  = $(LIB_LOCAL) -L$(NETCDF_LIB) -lnetcdff -lnetcdf $(NETCDF4_DEP_LIB)
 CPP1    = $(CPP) -P $(TRADFLAG)
 M4      = m4 -Uinclude -Uindex -Ulen
 AR      = ar
@@ -50,11 +50,7 @@ diffwrf:                diffwrf.F90
 	$(FC) -c $(FFLAGS) diffwrf.f
 	@if [ \( -f ../../frame/wrf_debug.o \) -a \( -f ../../frame/module_wrf_error.o \) -a \( -f $(ESMF_MOD_DEPENDENCE) \) -a \( -f ../../frame/clog.o \) ] ; then \
 	  echo "diffwrf io_netcdf is being built now. " ; \
-          if [ \( -f $(NETCDFPATH)/lib/libnetcdff.a -o -f $(NETCDFPATH)/lib/libnetcdff.so \) ] ; then \
-            $(FC) $(FFLAGS) $(LDFLAGS) -o diffwrf diffwrf.o $(OBJSL) ../../frame/wrf_debug.o ../../frame/module_wrf_error.o ../../frame/clog.o $(ESMF_IO_LIB_EXT) $(LIBFFS) ;\
-          else \
-            $(FC) $(FFLAGS) $(LDFLAGS) -o diffwrf diffwrf.o $(OBJSL) ../../frame/wrf_debug.o ../../frame/module_wrf_error.o ../../frame/clog.o $(ESMF_IO_LIB_EXT) $(LIBS) ;\
-          fi ; \
+          $(FC) $(FFLAGS) $(LDFLAGS) -o diffwrf diffwrf.o $(OBJSL) ../../frame/wrf_debug.o ../../frame/module_wrf_error.o ../../frame/clog.o $(ESMF_IO_LIB_EXT) $(LIBFFS) ;\
         else \
 	   echo "***************************************************************************** " ; \
 	   echo "*** Rerun compile to make diffwrf in external/io_netcdf directory         *** " ; \
