--- WRFV3/external/io_netcdf/makefile.netcdff	2008-07-21 14:34:50.000000000 -0600
+++ WRFV3/external/io_netcdf/makefile	2008-11-18 11:19:49.000000000 -0700
@@ -52,8 +52,8 @@
 	$(FC) -c $(FFLAGS) diffwrf.f
 	@if [ \( -f ../../frame/wrf_debug.o \) -a \( -f ../../frame/module_wrf_error.o \) -a \( -f $(ESMF_MOD_DEPENDENCE) \) ] ; then \
 	  echo "diffwrf io_netcdf is being built now. " ; \
-          if [ -f $(NETCDFPATH)/lib/libnetcdff.a ] ; then \
-            $(FC) $(FFLAGS) $(LDFLAGS) -o diffwrf diffwrf.o $(OBJSL) bitwise_operators.o ../../frame/wrf_debug.o ../../frame/module_wrf_error.o $(ESMF_IO_LIB_EXT) $(LIBS) -lnetcdff ;\
+          if [ -f $(NETCDFPATH)/lib/libnetcdff.so ] ; then \
+            $(FC) $(FFLAGS) $(LDFLAGS) -o diffwrf diffwrf.o $(OBJSL) bitwise_operators.o ../../frame/wrf_debug.o ../../frame/module_wrf_error.o $(ESMF_IO_LIB_EXT) $(LIBS) -lnetcdff `gcc -print-file-name=libgfortran.so` ;\
           else \
             $(FC) $(FFLAGS) $(LDFLAGS) -o diffwrf diffwrf.o $(OBJSL) bitwise_operators.o ../../frame/wrf_debug.o ../../frame/module_wrf_error.o $(ESMF_IO_LIB_EXT) $(LIBS) ;\
           fi ; \
